_non-cosmos/statemine/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "statemine",
+  "assets": [
+    {
+      "denom_units": [
+        {
+          "denom": "130",
+          "exponent": 0
+        },
+        {
+          "denom": "usdt",
+          "exponent": 6
+        }
+      ],
+      "base": "130",
+      "name": "Statemine",
+      "display": "usdt",
+      "symbol": "USDT",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "statemine",
```

Comment:
the bridge trace from itself (from statemine to statemine) doesn't make sense....

Reply:
ah updated